apps/ledger-live-mobile/src/components/DiscreetModeButton.tsx

```diff
@@ -4,13 +4,20 @@ import { useDispatch, useSelector } from "react-redux";
 import { EyeMedium, EyeNoneMedium } from "@ledgerhq/native-ui/assets/icons";
 import { discreetModeSelector } from "../reducers/settings";
 import { setDiscreetMode } from "../actions/settings";
+import { useCurrentRouteName } from "../helpers/routeHooks";
+import { track } from "../analytics";
 
 export default function DiscreetModeButton({ size = 24 }: { size?: number }) {
   const discreetMode = useSelector(discreetModeSelector);
+  const currentScreen = useCurrentRouteName();
   const dispatch = useDispatch();
   const onPress = useCallback(() => {
+    track("button_clicked", {
+      button: "Mask",
```

Comment:
Is `Mask` the right event name even when we switch the discreetMode from true to false ?

Reply:
I send now Mask / Unmask depending on discreetMode value